assets/schema_input.json

```
@@ -15,7 +15,7 @@
             },
             "assembly_type": {
                 "type": "string",
-                "enum": ["PRIMARY", "HAPLO", "MITO", "PLASTID"],
+                "enum": ["PRIMARY", "HAPLO", "HAP1", "HAP2", "MITO", "PLASTID"],
                 "errorMessage": "Not a valid assembly type, must be of [PRIMARY,HAPLOTYPE,MITO,PLASTID]"
```

```suggestion
                "errorMessage": "Not a valid assembly type, must be of [PRIMARY,HAPLO,HAP1,HAP2,MITO,PLASTID]"
```

But actually I think nf-schema will tell you this automatically without having to write your own error message

---

It would wouldn't it, that part of the point of the enum. I'll just delete the errorMessage.